components/rmrk/Profile/NavbarProfileDropdown.vue

```diff
@@ -135,7 +136,18 @@ export default class NavbarProfileDropdown extends mixins(PrefixMixin) {
     return this.$store.getters.getAuthAddress
   }
 
-  checkExtension() {
+  protected showRampSDK(): void {
+    new RampInstantSDK({
+      defaultAsset: 'KSM', // todo: prefix
+      userAddress: this.account,
+      hostAppName: 'KodaDot',
+      hostApiKey: 'a99bfvomhhbvzy6thaycxbawz7d3pssuz2a8hsrc', // env
+      hostLogoUrl: 'https://nft.kodadot.xyz/apple-touch-icon.png',
```

Comment:
```suggestion
      hostLogoUrl: 'https://kodadot.xyz/apple-touch-icon.png',
```
let's remove `nft.` prefix